_data/t.yml

```diff
@@ -196,4 +196,25 @@ LastUpdateDate:
   it: "Ultimo aggiornamento:"
 FormatDate:
   en: "%m/%d/%Y"
-  it: "%d/%m/%Y"
\ No newline at end of file
+  it: "%d/%m/%Y"
+SpeseDelTeam:
+  en: "Spese Team per la Trasformazione Digitale"
```

Comment:
Non so se ha senso linkare dalle pagine in inglese, non per ora perlomeno, il problema e' che diventa complesso tradurre i testi in `expenses.yml` - per ora metti pure il testo hardcoded e fai una pagina sola per l'italiano